Gemfile

```diff
@@ -20,11 +20,11 @@ if ENV['BEAKER_PE_PR_REPO_URL']
   gem lib, :git => "git@github.com:#{author}/#{lib}.git", :branch => ref
 end
 
-group :rubocop do
-  gem 'rubocop', '~> 1.12.0'
-  gem 'rubocop-performance'
-  gem 'rubocop-rake'
-  gem 'rubocop-rspec'
+group :tests do
+  gem 'rubocop', '~> 1.45.0'
+  gem 'rubocop-rspec', '~> 2.18.1'
+  gem 'rubocop-rake', '~> 0.6.0'
+  gem 'rubocop-performance', '~> 1.16.0'
```

Comment:
Previously the lines were sorted (and there's a cop that enforces it). Please keep that.

Reply:
done